### ISSUE TITLE
fix(tag): input is set on wrong version

### ISSUE THREAD

**Issue by devopsmariocom** [wc-catalogue/blaze-elements#236]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => You can submit support request here, but we preffer to chat about it realtime at our [ngParty slack](https://ngparty.herokuapp.com/)
```

**Current behavior**
tag has dependency on input ^1.0.0 which is outdated version and is breaking the build

detailed explanation:
- lerna download old npm package instead of linking
- package contain ts/tsx files which depends on _helpers folder as described #235

**Expected behavior**
we should have proper version

**Minimal reproduction of the problem with instructions**
`yarn test`

**Comment by Hotell** [comment]
Pls always remember to have Closes #NumberOfTicket in your commits, otherwise related issue wont be closed automatically.

thx!

**Comment by Hotell** [comment]
Closed by #237
